example/src/screens/calendars.js

```diff
@@ -107,7 +107,8 @@ const CalendarsScreen = () => {
             // textDisabledColor: 'red',
             'stylesheet.calendar.header': {
               week: {
-                marginTop: 5,
+                marginTop: 30,
+                marginHorizontal: 12,
```

Comment:
What is this for?

Reply:
Just a fix for the example, it looked really off